src/index.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './components/App/App';
+import { navigationList, state } from './Store/store';
+
+
+ReactDOM.render(
+  <React.StrictMode>
+    <App state={state} list={navigationList}/>
```

Comment:
Why are you passing this object through the whole tree? Just import what you need where you need.